setup.cfg

```diff
@@ -1,4 +1,5 @@
 [flake8]
+max-line-length: 120
```

Comment:
I think we should have uniform settings in brainiak and brainiak-extras. If you want to merge the code now and reformat it later, please exclude the files that fail the checks and make an issue to reformat them later.